preprocessors/content-categoriser/categoriser.py

```diff
@@ -83,7 +85,7 @@ def categorise():
         return "", 204  # No content
     request_uuid = content["request_uuid"]
     timestamp = time.time()
-    name = "ca.mcgill.a11y.image.preprocessor.firstCategoriser"
+    name = "ca.mcgill.a11y.image.preprocessor.content-categoriser"
```

Comment:
We typically use camel case for these.
```suggestion
    name = "ca.mcgill.a11y.image.preprocessor.contentCategoriser"
```